packages/react-server/src/ReactFlightServer.js

```diff
@@ -196,21 +200,26 @@ export function createRequest(
       'Currently React only supports one RSC renderer at a time.',
     );
   }
+  prepareHostDispatcher();
   ReactCurrentCache.current = DefaultCacheDispatcher;
 
   const abortSet: Set<Task> = new Set();
   const pingedTasks: Array<Task> = [];
+  const hints = createHints();
```

Comment:
Is this the "Hint" or is this more like the "Resources" since it's just for deduping, and the chunks are the hints?

Reply:
I could rename it but I suspect we'll need to make this thing actually hold the rows at some point. Then we can move the logic for which hints get emitted first into the DOM side and it can interleave the imports maybe.

I don't think of them as resources because it feels like that might be confused with Fizz resources which aren't really the same thing (also Fizz resources also intern their chunks so it's not really a pattern anywhere where we have just a pure cache.

I think for now it's pragmatic to just call it Hints.